client/components/date-range/shortcuts.tsx

```diff
@@ -59,7 +59,9 @@ const DateRangePickerShortcuts = ( {
 
 	const handleClick = ( shortcut: DateRangePickerShortcut ) => {
 		! locked &&
-			onClick( moment( shortcut.startDate ), moment( shortcut.endDate ), shortcut.id || '' );
+			shortcut.startDate &&
+			shortcut.endDate &&
+			onClick( moment( shortcut.startDate ), moment( shortcut.endDate ) );
```

Comment:
Another approach is to restore the selected start and end dates from the `DateRange` component by clicking on the `Custom Range` shortcut.